adhocracy4/comments/models.py

```diff
@@ -71,10 +71,10 @@ def save(self, *args, **kwargs):
             self.comment)
 
         if self.is_removed:
-            self.comment = _('deleted by creator')
+            self.comment = ('')
```

Comment:
As we have the same text for deleted and censored now, you can put it into the same if statement with an `or` and save a few lines. :)